src/arch/xtensa/configs/override/cannonlake_chrome.config

```diff
@@ -1,2 +1,3 @@
 CONFIG_CAVS_LPS=y
 CONFIG_CAVS_LPRO=y
+CONFIG_SAMPLES=y
```

Comment:
Shouldn't this be a "n" by default ? Since samples will be opt-in only if you are developing that feature.

Reply:
Sample code is a part of the delivery for chrome, therefore needs to be CI'ed. Not only compiled but tested to make sure there is no regression in the infrastructure supporting them.